### PR TITLE
N21-1236 add requirements to disable switch

### PR DESCRIPTION
**Pull request by IgorCapCoder**
# Short Description
the enableLdapSyncDuringMigration switch in the MigrationSection should be disabled when user login migration is not active

## Links to Ticket and related Pull-Requests
https://ticketsystem.dbildungscloud.de/browse/N21-1236
https://github.com/hpi-schul-cloud/schulcloud-server/pull/4373

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/components/administration/AdminMigrationSection.unit.ts [hunk 1]
```diff
@@ -783,8 +783,8 @@ describe("AdminMigrationSection", () => {
 			});
 		});
 
-		describe("when migration is not yet finished", () => {
-			it("should show switch button and description", () => {
+		describe("when migration is active", () => {
```
Comment (arnegns) [inline src/components/administration/AdminMigrationSection.unit.ts:786]: You have the "when migration is active" describe block two times

File: src/components/administration/AdminMigrationSection.vue [hunk 2]
```diff
@@ -131,6 +131,7 @@
 					'components.administration.adminMigrationSection.enableSyncDuringMigration.label'
 				)
 			"
+			:disabled="!oauthMigration.oauthMigrationPossible"
```
Comment (arnegns) [inline src/components/administration/AdminMigrationSection.vue:134]: Give this information to M.L.

**Review by arnegns — CHANGES_REQUESTED** [review]

File: src/components/administration/AdminMigrationSection.vue [hunk 3]
```diff
@@ -212,18 +213,19 @@ export default defineComponent({
...
-		const setMigration = (available: boolean, mandatory: boolean) => {
+		const setMigration = async (available: boolean, mandatory: boolean) => {
 			const migrationFlags: MigrationBody = {
 				oauthMigrationPossible: available,
 				oauthMigrationMandatory: mandatory,
 				oauthMigrationFinished: !available,
 			};
-			schoolsModule.setSchoolOauthMigration(migrationFlags);
+			await schoolsModule.setSchoolOauthMigration(migrationFlags);
+			await schoolsModule.fetchSchool();
```
Comment (arnegns) [inline src/components/administration/AdminMigrationSection.vue:223]: Did you test it?

Comment (IgorCapCoder) [reply src/components/administration/AdminMigrationSection.vue:223]: now i did ^^

**Review by arnegns — APPROVED** [review]

**Pull request merged**